maintenance/Rector/MockGrabyResponseRector.php

```diff
@@ -42,21 +42,9 @@ final class MockGrabyResponseRector extends AbstractRector
     private const FIXTURE_DIRECTORY = __DIR__ . '/../../tests/fixtures/content';
     private const MATCHING_ERROR_COMMENT = 'TODO: Rector was unable to evaluate this Graby config.';
     private const IGNORE_COMMENT = 'Rector: do not add mock client';
-    private BetterNodeFinder $betterNodeFinder;
-    private UseNodesToAddCollector $useNodesToAddCollector;
-    private ValueResolver $valueResolver;
-    private VariableNaming $variableNaming;
-
-    public function __construct(
-        BetterNodeFinder $betterNodeFinder,
-        UseNodesToAddCollector $useNodesToAddCollector,
-        ValueResolver $valueResolver,
-        VariableNaming $variableNaming
-    ) {
-        $this->betterNodeFinder = $betterNodeFinder;
-        $this->useNodesToAddCollector = $useNodesToAddCollector;
-        $this->valueResolver = $valueResolver;
-        $this->variableNaming = $variableNaming;
+
+    public function __construct(private readonly BetterNodeFinder $betterNodeFinder, private readonly UseNodesToAddCollector $useNodesToAddCollector, private readonly ValueResolver $valueResolver, private readonly VariableNaming $variableNaming)
```

Comment:
Would be more readable formatted like this (https://cs.symfony.com/doc/rules/function_notation/method_argument_space.html)


```suggestion
    public function __construct(
        private readonly BetterNodeFinder $betterNodeFinder,
        private readonly UseNodesToAddCollector $useNodesToAddCollector,
        private readonly ValueResolver $valueResolver,
        private readonly VariableNaming $variableNaming,
    )
```